app/assets/javascripts/enums.ts

```diff
@@ -22,3 +22,16 @@ export enum HtmlInputTypes {
   Url = 'url',
   Week = 'week'
 }
+
+export enum ErrorMessages {
```

Comment:
Nitpick but enums should typically be singular :) Same for HtmlInputTypes above but that's out of scope.

```suggestion
export enum ErrorMessage {
```

Reply:
In this case though it kind of reads weird when importing so maybe we can leave it as is. 

import { ErrorMessage } from '@/enums';

Reply:
Yeah, it really could be a bit confusing when is singular. But I guess it's better to have consistency and use best practices here, so I renamed it and HtmlInputTypes too, which I added in the last PR.